robotoff/spellcheck/patterns/__init__.py

```diff
@@ -0,0 +1,38 @@
+from typing import Dict
+from pathlib import Path
+
+from robotoff.spellcheck.base_spellchecker import BaseSpellchecker
+
+Patterns = Dict[str, str]
+
+
+class PatternsSpellchecker(BaseSpellchecker):
+
+    PATTERNS_PATHS = {"fr": Path(__file__).parent / "patterns_fr.txt"}
+
+    def __init__(self, lang: str = "fr"):
+        self.lang = lang
```

Comment:
A check to know if the lang is supported would be great here, especially as it could fail below with: `self.PATTERNS_PATHS[lang]`

Reply:
Added an exception here : https://github.com/openfoodfacts/robotoff/pull/181/commits/3f99d8b74f62c1ab9e03a422c6c7e05c68f68094
Error was silent since language is checked at an item level. We might want to change this behavior when we start to process multiple languages.